### DISCUSSION TITLE
10043润美国

### DISCUSSION THREAD

**Discussion by ljhOfGithub** [The-Run-Philosophy-Organization/run#39] [Q&A]
有操作吗

**Comment by the-run-philosophy** [comment]
这里有解答：https://github.com/The-Run-Philosophy-Organization/run/blob/main/%E6%B6%A6%E5%AD%A6%E5%AE%9E%E4%BE%8B/%E8%AE%A1%E7%AE%97%E6%9C%BA/%E4%B8%80%E4%BA%9B%E5%B8%B8%E8%A7%81%E7%9A%84%E8%AE%A1%E7%AE%97%E6%9C%BA%E5%8E%BB%E7%BE%8E%E5%9B%BD%E9%97%AE%E9%A2%98%E6%80%BB%E7%BB%93.md

**Comment by the-run-philosophy** [comment]
看 10043学校怎么润？这一部分

**Reply by the-run-philosophy** [reply]
更新内容： https://github.com/The-Run-Philosophy-Organization/run/blob/main/%E6%B6%A6%E5%AD%A6%E5%AE%9E%E4%BE%8B/10043%E5%AD%A6%E6%A0%A1%E5%A6%82%E4%BD%95%E6%B6%A6/10043%E6%B6%A6%E7%BE%8E%E6%96%B9%E6%B3%95.md
